packages/scandipwa/src/component/ProductActions/ProductActions.container.js

```diff
@@ -326,7 +342,27 @@ export class ProductActionsContainer extends PureComponent {
         } = customPrice;
 
         const selectedOptions = this.getSelectedOptions();
-        const prices = options.reduce((acc, { data = [] }) => {
+        const selectedOptionsMulti = this.getSelectedOptionsMulti();
+
+        const prices = options.reduce((acc, { data = [], option_id, type }) => {
+            /*
+            * Such types contain a single item within data
+            * as those are looked up on the option_id
+            */
+            const types = ['area', 'field', 'file'];
+            if (types.includes(type)) {
```

Comment:
`types` here is unnecessary variable. Let's keep the context clean and use the value directly: 
```js
if ([a, b, c].includes(...)) { ... }
```

Reply:
Acknowledged. Great advice tho!! :muscle: